### PR TITLE
PoC/service consumption

### PR DESCRIPTION
**Pull request by aburgosu**

### CODE REVIEW

File: src/main/java/com/fundation/logic/poc/FileUploader.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            } catch (IOException e) {
+                System.err.println("Unable to read file");
+                e.printStackTrace();
+            } finally {
+                try {
+                    if (fis != null) fis.close();
+                } catch (IOException e) {}
+            }
+        }
+
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/poc/FileUploader.java:55]: unnecessary empty line

File: src/main/java/com/fundation/logic/poc/PostService.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                }
+                EntityUtils.consume(resEntity);
+            } finally {
+                response.close();
+            }
+        } finally {
+            httpclient.close();
+        }
+    }
+
```
Comment (titor-oopart) [inline src/main/java/com/fundation/logic/poc/PostService.java]: unnecessary empty line

File: src/main/java/com/fundation/logic/poc/FileUploader.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
+package com.fundation.logic.poc;
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/poc/FileUploader.java:1]: add copyright tag

File: src/main/java/com/fundation/logic/poc/FileUploader.java [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.ClientProtocolException;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.mime.MultipartEntity;
+import org.apache.http.entity.mime.content.InputStreamBody;
+import org.apache.http.impl.client.DefaultHttpClient;
+import org.apache.http.params.BasicHttpParams;
+import org.apache.http.util.EntityUtils;
+
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/poc/FileUploader.java:15]: add documentation

File: src/main/java/com/fundation/logic/poc/ServiceConsumption.java [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class ServiceConsumption {
+    public static void main(String[] args) {
+        try {
+
```
Comment (MelissaRoman06) [inline src/main/java/com/fundation/logic/poc/ServiceConsumption.java:11]: unnecessary empty line

**Pull request merged**